### PR TITLE
Marine auto load

### PR DESCRIPTION
**Pull request by OH296**
- re-written the scr_start_load script
- refactored (slightly) the unit size script
- swapped all uses of armor to armour because it was pissing me off

### CODE REVIEW

**Comment by OH296** [comment]
@GalacticChimp i've tried to actually leave some comments in the scr_start_load to try and make it as intuitive. Let me know if it seems intuitive to you or not as i'm just really trying to make everything as clean and intuitive as i can.

File: objects/obj_controller/Create_0.gml [hunk 1]
```diff
@@ -529,7 +529,7 @@ exit_all=0;
 // 
 diplomacy=0;
 trading=0;
-trading_artifact=0;
+trading_artifat=0;
```
Comment (GalacticChimp) [inline objects/obj_controller/Create_0.gml:532]: this one was fine

Comment (OH296) [reply objects/obj_controller/Create_0.gml]: yeah sorted good catch not sure how that happened

**Review by GalacticChimp — CHANGES_REQUESTED** [review]
Looks fine, just please fix conflicts and retest

**Pull request closed without merging**